app/Jobs/RequeuePendingBatchesJob.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
naming nitpick: I think it would be slightly more helpful if the job name would contain a reference to the query service, something like `RequeuePendingQsBatchesJob` for example